### PR TITLE
Add changelog for 0.15

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: servant/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,137 @@
...
+  [servant-conduit](https://hackage.haskell.org/package/servant-conduit)
+  [servant-pipes](https://hackage.haskell.org/package/servant-pipes) and
+  [servant-machines](https://hackage.haskell.org/package/servant-machines)
+  respectively.
+
+  Writing new framing strategies is simpler. Check existing strategies for examples.
+
+  This change shouldn't affect you, if you don't use streaming endpoints.
+
+- *servant-client* Separate streaming client. [#1066](https://github.com/haskell-servant/servant/pull/1066)
```
Comment (alpmestan) [inline servant/CHANGELOG.md]: @domenkozar Does the following explanation make things clearer?

File: servant/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,137 @@
...
+  This change shouldn't affect you, if you don't use streaming endpoints.
+
+- Drop support for GHC older than 8.0
+  [#1008](https://github.com/haskell-servant/servant/pull/1008)
+  [#1009](https://github.com/haskell-servant/servant/pull/1009)
+
+- `ComprehensiveAPI` is a part of public API.
+  This API type is used to verify that libraries implement all core combinators. 
+  Now we won't change this type between major versions.
+  (This have been true for some time already).
```
Comment (alpmestan) [inline servant/CHANGELOG.md]: ha**s**

File: servant/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,137 @@
...
+  [#1070](https://github.com/haskell-servant/servant/pull/1070)
+
+- Remove `Servant.Utils.Enter` module
```
Comment (alpmestan) [inline servant/CHANGELOG.md]: Goodbye, old friend.

File: servant/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,137 @@
...
+  [#1064](https://github.com/haskell-servant/servant/pull/1064)
+
+- *servant-foreign* Add support so `HasForeign` can be implemented for
+  `MultipartForm` from [`servant-multipart`](http://hackage.haskell.org/package/servant-multipart)
+  [#1035](https://github.com/haskell-servant/servant/pull/1035)
+
+- *servant-server* Add `MonadMask Handler`
+  [#1068](https://github.com/haskell-servant/servant/pull/1068)
+
+- *servant-docs* Fix markdown indentatio
```
Comment (alpmestan) [inline servant/CHANGELOG.md]: missing **n** at the end

File: servant/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,137 @@
...
+      [#1007](https://github.com/haskell-servant/servant/pull/1007)
+    - miscellanea: fixed typos etc.
+      [#1030](https://github.com/haskell-servant/servant/pull/1030)
+      [#1020](https://github.com/haskell-servant/servant/pull/1020)
+      [#1059](https://github.com/haskell-servant/servant/pull/1059)
+
+- *Documentation* README
+  [#1010](https://github.com/haskell-servant/servant/pull/1010)
+
+- *servant-client-ghcjs* updates. **note** package is not release on Hackage
```
Comment (alpmestan) [inline servant/CHANGELOG.md]: release**d**

**Comment by domenkozar** [comment]
@phadej 

> You can use both.

Are you suggesting here to have separate API type and two clients? Imagine you have 200 routes and 5 of them are streaming, but they are scattered around your API tree. How do you use `runClientM` on 195 of the routes and `withClientM` on the 5 routes that use streaming?

Sorry for asking these questions, but I have that use case although I don't have 200 routes yet :)

**Comment by domenkozar** [comment]
I'm using `Servant.Generic` so regrouping is out of the question. I'm happy to ditch `servant-streaming` now that servant will have sufficient streaming support and that's why I looked into your work.

I'm fine rewriting my calls to `withClientEnv` or defining `runClientM` for myself. I'm also fine releasing 0.15 as-is, maybe I  should open a separate issue discussing if we can do better in order to introduce a streaming endpoints one has to rewrite the client api. Or at provide user guidance how to handle this situation as I'm in.

**Pull request merged**